openqa-schedule-mm-ping-test

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -eux -o pipefail
+
+openqa_url=${openqa_url:-https://openqa.opensuse.org}
+distri=${distri:-opensuse}
+flavor=${flavor:-DVD}
+arch=${arch:-x86_64}
+version=${version:-Tumbleweed}
+test_name=${test_name:-ping_client}
+
+tmpfile=$(mktemp)
```

Comment:
I suggest to use a cleanup jook, e.g.

```
trap 'rm $tmpfile' EXIT
```

And then without the rm-call after openqa-cli. No guarantee for first syntax though :)

Reply:
Yeah, that's a good idea